doc/source/whatsnew/v0.20.2.txt

```diff
@@ -53,6 +53,7 @@ Indexing
 I/O
 ^^^
 
+- Bug in skipping error lines with inline comments in space delimited text files (:issue:`16472`)
```

Comment:
something like this: Bug in ``pd.read_csv()`` when ``comment`` is passed in space deliminted text files.